samples/net/coap_client/src/coap-client.c

```diff
@@ -174,28 +214,17 @@ static void event_iface_up(struct net_mgmt_event_callback *cb,
 	struct net_pkt *pkt;
 	struct net_buf *frag;
 	int r;
-	u8_t observe = 0;
 
-	r = net_context_get(PF_INET6, SOCK_DGRAM, IPPROTO_UDP, &context);
-	if (r) {
-		printk("Could not get an UDP context\n");
-		return;
-	}
+	u8_t My_COAP_token1[8] = "puttok";
+	u8_t My_COAP_token2[8] = "posttok";
 
-	r = net_context_bind(context, (struct sockaddr *) &any_addr,
-			     sizeof(any_addr));
-	if (r) {
-		printk("Could not bind the context\n");
-		return;
-	}
+	u8_t put_payload[40] = "ZEPHYR COAP put method under test";
```

Comment:
Please do not put length when defining the array of this kind, just let the compiler calculate the length for you.